taskRequests/details/style.css

```diff
@@ -216,6 +224,61 @@ body {
   color: #c3c3c3;
   font-weight: 600;
 }
+.reject__container {
+  text-align: center;
+}
+.requestors__container__reject__button {
+  padding: 0.375rem 0.5rem;
+  width: 80%;
+  background: #fff;
+  border: solid 1px var(--color-red-variant1);
+  font-weight: 700;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: var(--color-red-variant1);
+  border-radius: 0.25rem;
+  cursor: pointer;
+}
+.requestors__container__reject__button:hover {
+  color: var(--color-white);
+  background: var(--color-red-variant1);
+  transition: 0.3s ease-in-out;
+}
+
+.circular-image {
+  border-radius: 50%;
+  max-width: 100%;
+  max-height: 100%;
```

Comment:
look for object-fit property, it works great with images we want to put in a container

Reply:
Tried `object-fit : fill , contain , cover` but none sees to be working .
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/97341921/0e9661c6-ddf2-4e2c-b499-ee6f1bb2b6d4)

Reply:
okay